conductor/src/main/java/com/bluelinelabs/conductor/internal/RouterRequiringFunc.kt

```diff
@@ -0,0 +1,9 @@
+package com.bluelinelabs.conductor.internal
+
+import androidx.annotation.RestrictTo
+import androidx.annotation.RestrictTo.Scope.LIBRARY_GROUP
+
+@RestrictTo(LIBRARY_GROUP)
+interface RouterRequiringFunc {
+    fun execute()
+}
```

Comment:
Can this be replaced by a typealias?

```kotlin
typealias RouterRequiringFunc = () -> Unit
```

Reply:
In the long term of cause but as it won't work with java interop, we an only replace this after all usages migrated to kotlin.